src/main/java/ai/rev/speechtotext/ApiInterface.java

```diff
@@ -54,4 +55,13 @@
           @QueryMap Map<String, String> query,
           @HeaderMap Map<String, String> contentType
   );
+
+  @POST("vocabularies")
+  Call<CustomVocabularyInformation> submitCustomVocabularies(@Body RevAiJobOptions options);
```

Comment:
There is no coupling between job options and cv options. This parameter shouldn't be the same RevAiJobOptions object. There should be a separate RevAiCustomVocabularyOptions definition. Some of the properties will have similar names, but this is fine because they are separate APIs with separate options.